sw/device/tests/mask_rom/mask_rom_fake_keys_sig_unittest.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/mask_rom/mask_rom_fake_keys_sig.h"
+
+#include "sw/device/lib/testing/mask_rom_test.h"
+#include "sw/device/tests/mask_rom/mocks/dif_fake_hmac_mock.h"
+#include "sw/device/tests/mask_rom/mocks/dif_fake_otbn_mock.h"
+
+namespace mask_rom_test {
+namespace {
+
+using namespace testing;
+using namespace mask_rom_test;
```

Comment:
Do not use using directives (i.e., `using namespace`). This breaks name resolution in weird ways, and is banned in Google C++.

See https://google.github.io/styleguide/cppguide.html#Namespaces.

(I know this is example code, but I feel it's very important to point out that using namespace is a gigantic footgun. https://abseil.io/tips/153 has some details.)